data/json/hobbies.json

```diff
@@ -707,7 +1190,14 @@
     "description": "You loved learning and sharing interesting facts about many different topics.  None of them are useful anymore, but your strong memory certainly is.",
     "points": 3,
     "skills": [ { "level": 1, "name": "speech" } ],
-    "traits": [ "GOODMEMORY" ]
+    "traits": [ "GOODMEMORY" ],
+    "difficulty": [
+      [ "combat", "normal" ],
+      [ "mobility", "normal" ],
+      [ "crafting", "normal" ],
+      [ "environment", "normal" ],
+      [ "social", "easy" ]
```

Comment:
```suggestion
      [ "social", "normal" ]
```

Reply:
One point in speech is not really enough, and good memory doesn't help with anything social as far as I know.